examples/quickstart/README.md

```diff
@@ -0,0 +1,10 @@
+# Quick Start Things
+
```

Comment:
Does it make sense to link to the possible consumer part? 
https://github.com/w3c/wot-cg/tree/main/Tutorials/quickstart

Reply:
I want to create it in ts and submit here but as a separate PR